testsuite/.rubocop_todo.yml

```diff
@@ -331,6 +333,7 @@ Style/AndOr:
     - 'features/step_definitions/file_management_steps.rb'
     - 'features/step_definitions/retail_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
Same for https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/AndOr if we can refactor whatever produce this alarm... it will be awesome.

Reply:
Fixed.